AnnoyancesFilter/sections/popups.txt

```diff
@@ -122,6 +122,8 @@ theoryandpractice.ru##a[href^="https://form.typeform.com/"]
 !
 softbank.jp##.mbox-banner
 shingakunet.com##.l-footerFix
+businesstoday.in##.bottomSliderSwipe
+businesstoday.in##.top_stories_breaking
```

Comment:
`businesstoday.in##.top_stories_breaking`
Is it reqlly annoyung? I see it on the main page at the top only.

Reply:
`businesstoday.in##.newsltter-iframe` already blocked by `Fanboy's Annoyances`

Reply:
It's a third party filter, you should block it in AdGuard Annoyances.